src/Layers/xrRenderPC_GL/gl_rendertarget_u_set_rt.cpp

```diff
@@ -27,8 +27,7 @@ void CRenderTarget::u_setrt(const ref_rt& _1, const ref_rt& _2, const ref_rt& _3
     }
     RCache.set_ZB(zb);
     //	RImplementation.rmNormal				();
-    GLenum status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
-    VERIFY(status == GL_FRAMEBUFFER_COMPLETE);
+    VERIFY(glCheckFramebufferStatus(GL_FRAMEBUFFER) == GL_FRAMEBUFFER_COMPLETE);
```

Comment:
Be cautious! VERIFY doesn't work on Release, while the logic of this code is that `glCheckFramebufferStatus` should be always executed, on Release too.
Correct change is this:
```suggestion
    [[maybe_unused]] GLenum status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
    VERIFY(status == GL_FRAMEBUFFER_COMPLETE);
```